modules/c/nrt/source/Utils.c

```diff
@@ -368,7 +368,7 @@ NRTAPI(NRT_BOOL) nrt_Utils_parseGeographicString(const char *dms, int *degrees,
     if (dir != 'N' && dir != 'S' && dir != 'E' && dir != 'W')
     {
         nrt_Error_initf(error, NRT_CTXT, NRT_ERR_INVALID_PARAMETER,
-                        "Invalid direction: %s. Should be [NSEW]", dms);
+                        "Invalid direction: %c. Should be [NSEW]", dir);
```

Comment:
If the DMS is messed up, I think it's useful to show the whole thing as there's probably a formatting bug (but I agree, the current exception is wrong).  How about adding something like  `"(Full DMS field is '%s')"`

Reply:
Added